static/src/stylesheets/atoms/vars.scss

```diff
@@ -9,5 +9,4 @@
     --f-serif-text: $f-serif-text;
     --f-serif-headline: $f-serif-headline;
     --f-sans-serif-text: $f-sans-serif-text;
-    --f-sans-serif-headline: $f-sans-serif-headline;
```

Comment:
What if you alias the text version instead of dropping the property? This way I won't block you while I check atoms aren't affected :-)

Reply:
is it a big check? i considered that as a general move, but didn't want to leave misleading indirection about. can do though if you need a little while to check?

Reply:
Perhaps cleaner if you just duplicated [here](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/atoms/vars.scss) and then remove the var above?

Reply:
... just checked, we're only using the text version. Then let's drop it altogether 👍

Reply:
woo! thanks for checking